salt/personalised-covers/config/etc-nginx-sites-enabled-api-dummy-dev.conf

```diff
@@ -0,0 +1,28 @@
+server {
+    listen 8080;
+
+    server_name localhost;
+
+    root /srv/api-dummy/web;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
```

Comment:
we shouldn't need these 2 `location` lines, port 8080 will be closed to the outside world anyway